com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/JavaDebugDelegateCommandHandler.java

```diff
@@ -32,6 +33,8 @@
 
     public static String CONFIG_LOG_LEVEL = "vscode.java.configLogLevel";
 
+    public static String UPDATE_USER_SETTINGS = "vscode.java.updateUserSettings";
```

Comment:
We need to not confuse the user by providing the command vscode.java.updateUserSetting. It's actually debugger settings. So suggest making it "vscode.java.updateDebuggerSettings".